gpt_index/indices/keyword_table/base.py

```diff
@@ -62,7 +62,7 @@ def __init__(
             chunk_size=chunk_size,
             chunk_overlap=MAX_CHUNK_OVERLAP,
         )
-        super().__init__(documents=documents, index_struct=index_struct)
+        super().__init__(documents=documents, index_struct=index_struct, **kwargs)
```

Comment:
For my understanding only: why did we make this a general `kwargs`? From what i can tell, in the `BaseGPTIndex` constructor we just added `llm_predictor`. https://github.com/jerryjliu/gpt_index/blob/main/gpt_index/indices/base.py#L58

Reply:
Yeah it was mostly because I was lazy, in case any additional arguments popped up...I didn't lump documents/index_struct into kwargs because they're in first position in the arguments.

Actually probably not a bad idea to be explicit instead of using kwargs here. I'll take a look at cleaning this up as an additional TODO